c_tests/tests/compile_calls.c

```diff
@@ -11,21 +11,24 @@
 #include <string.h>
 #include <yk_testing.h>
 
-void f() { int b = 1; }
+int f() {
```

Comment:
It's probably worth marking this function as "don't align" -- it's not *guaranteed* to work, but it does make it harder for the optimiser to inline it!

Reply:
Fixed in 14dca0be.